utils/src/commonMain/kotlin/com/badoo/reaktive/utils/Freeze.kt

```diff
@@ -3,5 +3,8 @@ package com.badoo.reaktive.utils
 expect fun <T> T.freeze(): T
 
 expect fun <T: Any> T.ensureNeverFrozen(): T
+<<<<<<< HEAD
```

Comment:
Could you please resolve all the conflicts?

Reply:
Yes, I have 👍